test/adapters/adapters.spec.js

```diff
@@ -10,22 +11,31 @@ let retry = require('bluebird-retry');
 let tmp = require('tmp');
 let uuid = require('uuid');
 
-function juttleBin(juttle) {
+function juttleBin(juttle, options) {
+    options = _.defaults(options, {
+        quiet: true
+    });
     logger.log(`juttle -e "${juttle}"`);
     return docker.exec('juttle-engine-local',
                        ['/opt/juttle-engine/bin/juttle',
                         '-e',
-                        juttle], { quiet: true });
+                        juttle], options);
 }
 
 describe('adapters', () => {
 
+    let tmpdir;
+
     before(() => {
-        var tmpdir = tmp.dirSync().name;
+        tmpdir = tmp.dirSync().name;
+
         return Promise.all([
             docker.destroy('elasticsearch-adapter-test'),
             docker.destroy('influxdb-adapter-test'),
             docker.destroy('graphite-adapter-test'),
+            docker.destroy('mysql-adapter-test'),
+            docker.destroy('postgres-adapter-test'),
+            docker.destroy('sqlite-adapter-test'),
```

Comment:
Since you're just creating a sqlite db within the juttle-engine container, I don't think there's a sqlite-adapter-test container to destroy, right?